### DISCUSSION TITLE
Resilience dependencies for ORAS

### DISCUSSION THREAD

**Discussion by sajayantony** [oras-project/oras-dotnet#73] [General]
Enabling a good foundation, as this library matures and as we add more resilience capabilities to this like #49, would be critical to the long term usage. 

With that in mind and having had some discussions internally with folks like @kichalla and @nathana1, I would like to propose we align with dotnet 8.0 which is LTS and has good alignment with [Polly](https://www.thepollyproject.org/). 

The main advantages I see are that we consider things like retries and resiliency from the get-go before we create an auth model. This also enables investing in testing as well. 

The downsides, which all choices usually have, are that we have a larger dependency footprint. However, the fact remains that we create a library that can someday be used in serious application and not just for a sample or demo. 

Thoughts? @shizhMSFT @Wwwsylvia

**Comment by shizhMSFT** [comment]
Sounds a good idea. According to the [official .NET doc](https://learn.microsoft.com/dotnet/standard/net-standard?tabs=net-standard-1-0#when-to-target-net80-vs-netstandard), new libraries should target `net8.0`.

**Comment by shizhMSFT** [comment]
Closed by #75
